doc/GregorioNabcRef.tex

```diff
@@ -89,6 +89,246 @@
 }
 
 {
+\setlength\parindent{0cm}
+
+The \texttt{nabc} language allows to describe some adiastematic neumes, for
+now just the St. Gallen style.  The language is partially based on Dom
+Eugène Cardine's Table of neumatic signs, but for more complex neumes
+doesn't always match how the neumes are called; instead attempts to make it
+easier to compose complex neumes from basic glyphs.  To describe
+adiastematic neumes in \texttt{gabc}, the header should contain
+\texttt{nabc-lines: 1;} line, like:
+
+\begin{verbatim}
+nabc-lines: 1;
+(f3) AL(ef~|ta>)le(fg/hggf|peclhgpi)lú(ef~|ta>)ia.(f.|ta-) (,)
+(ii//|bv-|gh!ivHG//|vi-hhppu2su1sut1|fhg/|to|eef.|pt) (;)
+\end{verbatim}
+
+The \texttt{nabc} snippets are then separated by \texttt{|} character from
+\texttt{gabc} snippets.  Every \texttt{gabc} snippet may be followed by
+up to \texttt{nabc-lines} \texttt{nabc} snippets, followed optionally
```

Comment:
This sentence is slightly confusing. May be better as:

Every \texttt{gabc} snippet may be followed by multiple \texttt{nabc} snippets. The maximum number of \texttt{nabc} snippets is the number declared in the header field \texttt{nabc-lines: x;}. More \texttt{gabc} snippets can follow a \texttt{nabc} snippet group in an alternating pattern. (gabc|nabc|gabc|nabc|...etc...)